Source/GNUmakefile

```diff
@@ -487,6 +499,7 @@ GSPrintOperation.h \
 GSEPSPrintOperation.h \
 GSPDFPrintOperation.h \
 GSModelLoaderFactory.h \
+GSImageMagickImageRep.h \
```

Comment:
This file is no longer needed externally, but also no harm in copying it there.

Reply:
I figure it is useful.  I believe not including it there was an oversight.

Reply:
agreed